protected/humhub/modules/comment/models/Comment.php

```diff
@@ -300,4 +322,16 @@ public function isUpdated()
     {
         return $this->created_at !== $this->updated_at && !empty($this->updated_at) && is_string($this->updated_at);
     }
+
+    /**
+     * Checks if given content object is a subcomment
+     *
+     * @param $object
+     * @return bool
+     * @since 1.8
+     */
+    public static function isSubComment($object)
```

Comment:
We need to make sure to not collide with https://github.com/humhub/humhub/pull/4571/files#diff-4fcd54238adfe8774ee607eeadd9ff1c821166188d5f9dabd37cac4e2729a218R311

Reply:
Done! Remove this method